docs/user/README.md

```diff
@@ -16,9 +16,9 @@ The following commands are for Ubuntu Server. However, equivalent commands shoul
 1. Install [cURL](https://curl.se/) as the *root* user: `pkg install curl`
 
 ### Windows
-1. Install a supported version of [Visual Studio](https://visualstudio.microsoft.com/vs/older-downloads/) with Visual C++:
-   - [Visual Studio 2019](https://visualstudio.microsoft.com/vs/older-downloads/#visual-studio-2019-and-other-products) ([Supported](https://github.com/microsoft/snmalloc/blob/main/docs/BUILDING.md#building-on-windows))
-   - [Visual Studio 2022](https://visualstudio.microsoft.com/vs/) ([Supported](https://github.com/microsoft/snmalloc/blob/main/docs/BUILDING.md#building-on-windows))
+1. Install the Microsoft Visual C++ Prerequisites for Rust and Python 3:
+   - Microsoft Visual C++ Prerequisites for Rust: see [The rustup book](https://rust-lang.github.io/rustup/installation/windows-msvc.html).
+   - Python 3: install [Python 3](https://www.python.org/) and select _"Add python.exe to PATH"_.
```

Comment:
is any version of Python 3 fine or should we be more specific?

Reply:
The documentation for `maturin` states that it works for [CPython3.5+](https://github.com/PyO3/maturin) and both [CPython 3.5 and CPython 3.6](https://devguide.python.org/versions/) is already end-of-life, so any version of CPython 3 that is still supported should be fine. So I think it is fine to not mention a specific CPython version as we support all versions that is not end-of-life. What do you think @chrthomsen?

Reply:
Maybe we could simply add something like "the latest version" to Line 31 so it becomes "1. Install the latest version of the Microsoft Visual C++ Prerequisites for Rust and Python 3 "?

Reply:
Agreed again